### DISCUSSION TITLE
[Feature] Partner photos also visible on Map, available for search and available in the timeline's memory feature, also search for duplicates accross partners

### DISCUSSION THREAD

**Discussion by jazzdd86** [immich-app/immich#7427] [Feature Request]
### The feature

The app and its functionalty is awesome. Though, there are a few things I am missing:
It should be possible (maybe as an option) to include all partner's photos in the map view, as searchable assests and also in the timeline's memory view. As of now, only my photos are visible in the memory function/map even if there are many more photos taken by the partner.

There is also missing a function to search for duplicates accross partners - sometimes a partner has send a photo via messenger and it will then be uploaded by both partners. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #5457,  #6339
